src/arrayMethodIndexOf.test.js

```diff
@@ -14,37 +14,43 @@ test(`indexOf2 doesn't call default indexOf`, () => {
     .toBe(false);
 });
 
-test(`without valueToFind`, () => {
+test(`should return -1 when the search element is not specified`, () => {
   expect(source.indexOf2())
     .toBe(-1);
 });
 
-test(`indexOf2('11') = 0`, () => {
+test(`should return elements index when it is present`, () => {
   expect(source.indexOf2('11'))
     .toBe(0);
 });
 
-test(`indexOf2(11) = -1`, () => {
+test(`should return -1 when element of a different type is present`, () => {
   expect(source.indexOf2(11))
     .toBe(-1);
 });
 
-test(`indexOf2('1') = -1`, () => {
+test(`should return -1 when only substring of the element is present`, () => {
   expect(source.indexOf2('1'))
     .toBe(-1);
 });
 
-test(`indexOf2('11', 1) = -1 fromIndex 1`, () => {
+// eslint-disable-next-line max-len
```

Comment:
You pretty much never want to disable eslint errors, eslint enforces good practices which make the code more readable. 

Just add a line break to make this line shorter.

Reply:
done